File: pinyougou-web-shop/src/main/webapp/js/service/uszerService.js
//服务层
app.service('uszerService',function($http){
	    	
	//读取列表数据绑定到表单中
	this.findAll=function(){
		return $http.get('../uszer/findAll.do');		
	}
	//分页
	this.findPage=function(page,rows){
		return $http.get('../uszer/findPage.do?page='+page+'&rows='+rows);
	}
	//增加 
	this.add=function(entity){
		return  $http.post('../uszer/add.do',entity );
	}
	//修改 
	this.update=function(entity){
		return  $http.post('../uszer/update.do',entity );
	}

});

File: pinyougou-interface/src/main/java/cn/itcast/core/service/uszer/UszerService.java
package cn.itcast.core.service.uszer;

import cn.itcast.core.entity.PageResult;
import cn.itcast.core.pojo.uszer.Uszer;

import java.util.List;

public interface UszerService {


     // 用户注册
    public void add(Uszer uszer);


    //回显用户信息!!!!z
    public Uszer findOne(Integer uszerId);


    public Uszer findOneByUsername(String username);


    //查询所有用户
    List<Uszer> findAll();

    //更新
    public void update(Uszer uszer);

    //分页查询
    PageResult findPage(Integer pageNo, Integer pageSize);
}
